hamilton/graph.py

```diff
@@ -894,7 +899,7 @@ def display(
             custom_style_function=custom_style_function,
             config=config,
         )
-        kwargs = {"view": False, "format": "png"}  # default format = png
+        kwargs = {"format": "png"}  # default format = png
```

Comment:
why drop `"view": False`?

Reply:
`graphviz.Digraph.pipe()` doesn't have a `view` kwarg. Forgot to add it back for `keep_dot=True` though.

Will try to investigate the failed CI for `build-py38`